clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -786,6 +786,7 @@ def write_bids(
                 metadata.run_num,
             )
     correct_fieldmaps_name(to)
+    delete_real_imaginary_files(to)
```

Comment:
What is the consequence of deleting these files ? Is the output still valid ? BIDS-compliant ?

Reply:
This makes the ouput BIDS compliant, as `sub-x_ses-y_dwi_real.nii.gz is not.

Reply:
Thanks. What I'm not sure to understand is: 

- Is there also a `sub-x_ses-y_dwi.nii.gz`  (in addition to its `real` and/or `imaginary` versions) ?
- If not, this means we are removing all dwi images for these cases. Is this the best strategy ?

Reply:
in this specific case there is no `sub-x_ses-y_run-w_dwi`. Because, yes, since this is `Philips`, it would only be a run amongst others. That makes the other runs unusable since you need all of them. and it would break BIDS compliance. 
For these reasons, i think it is the best strategy !

Reply:
Thanks for the explanations @MatthieuJoulot !